gnucash/gnome/window-reconcile.c

```diff
@@ -1200,6 +1194,12 @@ gnc_reconcile_window_create_view_box(Account *account,
     gtk_box_pack_start(GTK_BOX(hbox), label, FALSE, FALSE, 0);
     *total_save = label;
 
+#if GTK_CHECK_VERSION(3,12,0)
+    gtk_widget_set_margin_end (GTK_WIDGET(label), 10);
+#else
+    gtk_widget_set_margin_right (GTK_WIDGET(label), 10);
+#endif
+
```

Comment:
I only see now this is hardcoded to 10px. How does this work out with themes that use different margins ? Or are all the relevant margins hardcoded by us in this area ?

Reply:
We are specifying them in gnc_reconcile_window_create_view_box of window-reconcile.c

Reply:
Ok, then they remain in sync.